src/Core/Console.php

```diff
@@ -37,6 +37,7 @@ protected function getHelp()
 	config                 Edit site config
 	contact                Contact management
 	createdoxygen          Generate Doxygen headers
+	daemon                 Interacting with the Friendica daemons
```

Comment:
I don't believe we are breeding daemons at Friendica, we only have that trusty one.
```suggestion
	daemon                 Interact with the Friendica daemon
```

Reply:
good catch .. I thought about moving jetstream to this command as well - so one daemon - many daemons, isn't it? hahaha